packages/atomic/src/components/atomic-result-template/atomic-result-template.tsx

```diff
@@ -6,6 +6,9 @@ import {
 } from '@coveo/headless';
 import {MapProp} from '../../utils/props-utils';
 
+/**
+ * This doesn't seem to ever be used. Can I just delete?
```

Comment:
Since this interface is exported I wanted to document it, but in trying to figure out what it's for I realized that it's never used. Should I just delete it while I'm here?

Reply:
Yes thank you!